### ISSUE TITLE
Is it possible to hide launcher when icon is clicked

### ISSUE THREAD

**Issue by abelal** [alamminsalo/qml-launcher#3]
Is it possible that rather than closing (Qt.quit()) the launcher when one of the applications are selected we can simply hide the launcher and show it again when the launched application finishes? I understand that we can use waitForFinished in the process.cpp to at least wait for the subprocess but I am not able to hide/show the launcher...

**Comment by alamminsalo** [comment]
Yes, it would be trivial to accomplish this using a lockfile or something similar for single instance solution (Even dbus messaging?). It would be good to investigate how other launchers have this implemented. 

For now, I've decided to keep this clean and simple but incase any poor soul likes to tinker around I'd be happy to take PR.
